### PR TITLE
Blog app: add devise

### PR DESCRIPTION
**Pull request by tafaramafemba**
### Hello 👋, In this pull request we managed to:

- Remove current_user method in ApplicationController.
- Install and setup devise
- Modify the views in devise for registration and login to match the wireframe and styling.

### CODE REVIEW

File: db/migrate/20220719155700_add_devise_to_users.rb [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      ## Rememberable
+      t.datetime :remember_created_at
+
+      ## Trackable
+      # t.integer  :sign_in_count, default: 0, null: false
+      # t.datetime :current_sign_in_at
+      # t.datetime :last_sign_in_at
+      # t.string   :current_sign_in_ip
+      # t.string   :last_sign_in_ip
+
```
Comment (uwadonat) [inline db/migrate/20220719155700_add_devise_to_users.rb:23]: - [x] I am facing the following error while running migration. Kindly fix it.
Screenshot from 2022-07-20 02-50-01

**Review by uwadonat — CHANGES_REQUESTED** [review]
## Hi @Team,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Your pull request message is  very descriptive ✔️
- You added professional readme file ✔️
- No linter errors ✔️
almost



## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20220719155700_add_devise_to_users.rb (hunk 1 above)
Comment (tafaramafemba) [reply db/migrate/20220719155700_add_devise_to_users.rb:23]: Hello @uwadonat, the add_user_ref_to_posts.rb migration is not part of the project requirements. I have implemented the requirements that were given by Microverse. The migration you have attempted to run was run in the project that was 2 weeks ago. That was already approved and the relation exists.

The highlighted migration has already worked on my PC and I am able to log in and out as well as create a user account. I have attached screenshots

Comment (Mihndim2020) [reply db/migrate/20220719155700_add_devise_to_users.rb:23]: Hello @tafaramafemba, @uwadonat, this command should fix the issue: `rails db:reset db:create db:migrate db:seed
`. The project runs normally after updating your database.yml file accordingly.

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @tafaramafemba, 

### Good job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Clean/Organized code
:heavy_check_mark: Your project is running as expected

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20220719155700_add_devise_to_users.rb (hunk 1 above)
Comment (uwadonat) [reply db/migrate/20220719155700_add_devise_to_users.rb:23]: Nice! Thanks.

**Pull request merged**